docs/source/deployment.rst

```diff
@@ -0,0 +1,4 @@
+Deployment
+==========
+
+.. include:: ../../birdhouse/README.rst
```

Comment:
Oh neat, that's how Sphinx can refer to files outside of the `docs` folder?  Awesome, I didn't know this !  

With this trick we can get the best of both worlds: in place docs when browsing on github and dedicated RtD site.